package.json

```diff
@@ -29,6 +29,7 @@
   "dependencies": {
     "@google-cloud/logging": "^1.2.0",
     "anymatch": "^1.3.2",
+    "axios": "^0.18.0",
```

Comment:
Don't think these are needed, but maybe I forgot something.

Reply:
Yeah it may just be my machine, but I can't seem to build without `axios` and `grpc` 😔